tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -163,17 +163,16 @@ $variable =
         (
             is_array($this->modelData) || $this->modelData instanceof \\Countable
         ) &&
-        0 === count($this->modelData)
+            0 === count($this->modelData)
     ) ||
     // traversables that are not countable
     (
         $this->modelData instanceof \\Traversable &&
-        0 === iterator_count($this->modelData)
+            0 === iterator_count($this->modelData)
     );
 
 if (
-    2 & /* MB_OVERLOAD_STRING */
-    (int) ini_get('mbstring.func_overload')
+    2 & (int) ini_get('mbstring.func_overload') /* MB_OVERLOAD_STRING */
```

Comment:
I couldn't seem to figure this one out - I think there enough improvements though its worth pushing this and opening up an issue for this case with the comment

Reply:
:+1: agree